### PR TITLE
Remove -Xcheck:jni from junit JVM options

### PR DESCRIPTION
**Pull request by wolffcm**
It slows down execution too much on Java 8.  There are also lots of
JNI warnings that are issued when we run with Java 8.  I'll file a
separate ticket to address them.

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -1690,7 +1690,6 @@ TEST CASES
         >
             <classpath refid='project.classpath' />
             <jvmarg value="-server" />
-            <jvmarg value="-Xcheck:jni" />
```
Comment (jxchinaren) [inline build.xml]: Is it reasonable to have a comment here and comment out the -Xcheck:jni flag other than delete it completely? I wonder someone in future will add it back without any notice.

This is my ONLY comment.

Comment (wolffcm) [inline build.xml]: Sure, I'll add a comment explaining why.

**Comment by jxchinaren** [comment]
Looks good to go.

**Comment by akhanzode** [comment]
Do we really want to do this? If yes I would still create a target that does check:jni and let jenkins do that in addition.

**Comment by wolffcm** [comment]
@akhanzode I filed ENG-8898 for running JUnit tests in Jenkins with check:jni turned on for selected jobs.  I don't think we'll want it turned on normally if we ever switch to Java 8 by default---the test I looked at took five times longer under Java 8 with check:jni turned on.

https://issues.voltdb.com/browse/ENG-8898

Does this ticket seem sufficient to you?  If you still have concerns, let's figure what would work best.

**Comment by akhanzode** [comment]
That ticket works for me. I hope it gets done and doesn't get prioritized down.

**Pull request merged**